pkgs/applications/networking/browsers/palemoon/default.nix

```
@@ -45,15 +45,15 @@ assert with lib.strings; (
 
 stdenv.mkDerivation rec {
   pname = "palemoon";
-  version = "31.0.0";
+  version = "31.1.0";
 
   src = fetchFromGitea {
     domain = "repo.palemoon.org";
     owner = "MoonchildProductions";
     repo = "Pale-Moon";
     rev = "${version}_Release";
     fetchSubmodules = true;
-    sha256 = "sha256-fIQAQCtjA/9Otft3e9Z4xWgE09sqsdArYQtZqmEgfTc=";
+    sha256 = "sha256-82OvD4q89D0rxMdzPEabMGLXXtEt4A7P6p2gKi7o+oA=";
```

```suggestion
    sha256 = "sha256-x3n4OeZbnJCPCVjsZJW1nBYlsEYn6fXt80voYWQSNq4=";
```

Changed sha256 for the above rev change. Retested with that rev & still working. Someone lmk if the code suggestions aren't fine for commit-on-merge & automatic backport, then I'll apply them in an extra PR.